### PR TITLE
QUAKEY

### PR DESCRIPTION
**Pull request by eROADZ**
# Validate [QUAKEY](https://solscan.io/token/EpvwZ75jycGTU8w3qQgq8Ws1uUZsXXZSJJYW6B9gpump)

## Attestations (Please provide links):
- Tweet from your Twitter Account attesting the Mint address, tagging [@JupiterExchange](https://twitter.com/JupiterExchange) and showing community support: https://twitter.com/QUAKEYOFFICIALX/status/1804652297697259628
- Coingecko/ CMC URL (If available): https://www.coingecko.com/en/coins/YOUR_ID_HERE

## Validation (Please check off boxes):
- [x] The metadata provided in the PR matches what is on-chain (Mandatory)
- [x] Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
- [ ] Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)  

## Acknowledgement (Please check off boxes)
- [x] My change matches the format in the file (no spaces between fields).
- [x] My token is already live and trading on Jupiter.
- [x] !!! I read the README section on Community-Driven Validation and understand this PR will be only be reviewed when there is community support on Twitter.
- [x] Please make sure your pull request title has your token name. If it just says "Main", or "Validate", it will automatically be closed. PRs containing broken attestation or solscan links will also be closed.

### CODE REVIEW

**Review by eROADZ — COMMENTED** [review]
Love this project I was brought to help the project and it truly has changed my life many new things ahead like NFT lines and many other.

**Review by eROADZ — COMMENTED** [review]
This coin will be the next coin among the great I believe it.

File: validated-tokens.csv [hunk 1]
```diff
@@ -1794,3 +1794,4 @@ Mechelle Obema,OBEMA,CzkM8bzWFdXsjtZQnz2piTxJKPtJ5mfTL8S6sNZg7n7S,2,https://stat
 BILLY,BILLY,3B5wuUrMEi5yATD7on46hKfej3pfmd7t1RKgrsN3pump,6,https://cf-ipfs.com/ipfs/QmdRU5EkGt1VsgLsXJdHK5kjojQqggQHjhyqeJwmApR53R,true
 Genie,Genie,3Jjt8QhbqNoYfSQYHWf8ZsTJwE2CyvmUrzgzJD5Jpump,6,https://pbs.twimg.com/profile_images/1801011420525465600/2ywH00FN_400x400.jpg,true 
 Monkey Gold,MOGO,2WUecdL1GQA6pwQs7sMsCf2shEfZxWK2RaNHkthmLtmK,9,https://arweave.net/w_cYAcuwGOKxZWafjj-o14Ab_0PaaA_fZoQ8wqdy4rc,true
+QUAKEY,QUAKEY,EpvwZ75jycGTU8w3qQgq8Ws1uUZsXXZSJJYW6B9gpump,6,https://image-cdn.solana.fm/images/?imageUrl=https://cf-ipfs.com/ipfs/QmZHWA4nX3iMxiWGqQKdFcEvSogLkJp6pDJfckmp8spZ6z,true
```
Comment (eROADZ) [inline validated-tokens.csv:1797]: Love this project anyone else think so?

**Comment by Abstorm212** [comment]
QUAKEY is hands down the most legit crypto project in the memecoin space, having real utility, and the only memecoin to have utility. Everyone needs to know about this. This is the next multi billion dollar market cap coin.

**Comment by cwade77** [comment]
This coin deserves to be listed. The entire project is the most viable longterm memecoin in the space. It has major potential to be a true leader in the crypto industry.

**Comment by Iamaliny** [comment]
This is truly an amazing project, community driven, transparent, and tied to some big things. Let's Gooooo.

LONG ON $QUAKEY TO THE MOON

🚀🚀🚀

#IYKYK 😎

**Comment by Jymfletzfur** [comment]
token is too new, please come back in 10 days

**Pull request closed without merging**